src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.GlobalSuppressMessageFixAllCodeAction.cs

```diff
@@ -124,7 +125,7 @@ protected override async Task<Document> GetChangedSuppressionDocumentAsync(Cance
                 var services = suppressionsDoc.Project.Solution.Workspace.Services;
                 var suppressionsRoot = await suppressionsDoc.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
                 var addImportsService = suppressionsDoc.GetRequiredLanguageService<IAddImportsService>();
-                var options = await SyntaxFormattingOptions.FromDocumentAsync(suppressionsDoc, cancellationToken).ConfigureAwait(false);
+                var cleanupOptions = await CodeCleanupOptions.FromDocumentAsync(suppressionsDoc, fallbackOptions: null, cancellationToken).ConfigureAwait(false);
```

Comment:
taken offline, but what's the logic we're using to determien if there are fallbackOptions or not?

Reply:
Currently I have not passed fallback options in all places that should have them to avoid bloating the PR. Will be done in next PR.